cpp/oneapi/dal/algo/knn/backend/gpu/infer_kernel_brute_force_dpc.cpp

```diff
@@ -75,22 +90,36 @@ class knn_callback {
               result_options_(results),
               query_block_(query_block),
               query_length_(query_length),
-              k_neighbors_(k_neighbors),
-              voting_(pr::make_uniform_voting(q, query_block, k_neighbors)) {
+              k_neighbors_(k_neighbors) {
         if (result_options_.test(result_options::responses)) {
             this->temp_resp_ = pr::ndarray<idx_t, 2>::empty(q,
                                                             { query_block, k_neighbors },
                                                             sycl::usm::alloc::device);
         }
     }
 
+    auto& do_sqrt(bool do_sqrt) {
```

Comment:
Can this be named more clear? E.g., why in some cases we need to do apply `sqrt` - can we name the parameter according to the user needs?